ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/StateManager.java

```diff
@@ -511,6 +511,15 @@ public final Uid get_uid ()
         return objectUid;
     }
 
+    /**
+     * @return the object's birth date
```

Comment:
Will you indicate the time units stating that it returns the time the object was created in units of milliseconds since midnight, January 1, 1970 UTC (and include a reference to @see <a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/System.html#currentTimeMillis()">System.currentTimeMillis</a>) or something similar

Reply:
Javadoc is updated